ipaplatform/base/paths.py

```diff
@@ -79,6 +80,9 @@ class BasePathNamespace:
     LDAP_CONF = "/etc/ldap.conf"
     LIBNSS_LDAP_CONF = "/etc/libnss-ldap.conf"
     NAMED_CONF = "/etc/named.conf"
+    NAMED_CUSTOM_CONFIG = "/etc/named/ipa-ext.conf"
+    NAMED_CUSTOM_CFG_SRC = os.path.join(USR_SHARE_IPA_DIR,
+                                        'bind.ipa-ext.conf')
```

Comment:
This will not work as you expect.
Imagine that a some platform overrides USR_SHARE_IPA_DIR to `/foo/bar`, but NAMED_CUSTOM_CFG_SRC remains as it was defined in base paths (`/usr/share/ipa/bind.ipa-ext.conf`), not the expected `/foo/bar/bind.ipa-ext.conf`.
That's why there are just plain paths.

Reply:
hm.. ok. not really clean imho, but since other paths are also plain in there...